heron/executor/src/python/heron_executor.py

```diff
@@ -536,6 +546,79 @@ def prepare_launch(self):
         do_print("Failed to run command: %s. Exiting" % command)
```

Comment:
`do_print` is changed to `Log`. See: https://github.com/twitter/heron/pull/1252/files

Rebasing against master should fix this.